ui/app/src/state/epics/configuration.ts

```diff
@@ -46,13 +56,28 @@ export default [
         )
       )
     ),
-  (action$, state$) =>
+  (action$, state$, { getIntl }) =>
     action$.pipe(
       ofType(fetchSiteConfig.type),
       withLatestFrom(state$),
       filter(([, state]) => Boolean(state.sites.active)),
       exhaustMap(([, state]) =>
         fetchSiteConfigService(state.sites.active).pipe(
+          tap((config) => {
+            try {
+              let localeCode = config.locale?.localeCode || state.uiConfig.locale.localeCode;
+              let options = config.locale?.dateTimeFormatOptions || state.uiConfig.locale.dateTimeFormatOptions;
+              new Intl.DateTimeFormat(localeCode, options);
+            } catch (e) {
+              const hostToHost$ = getHostToHostBus();
+              hostToHost$.next(
+                showSystemNotification({
+                  // @ts-ignore
+                  message: getIntl().formatMessage(configurationMessages.localeError, { message: e.message })
+                })
+              );
+            }
+          }),
           map((config) => fetchSiteConfigComplete(config)),
```

Comment:
Why ts-ignore? Should need to be one. **ALWAYS** explain any ts/eslint ignores you add.
```suggestion
          switchMap((config) => {
            try {
              let localeCode = config.locale?.localeCode || state.uiConfig.locale.localeCode;
              let options = config.locale?.dateTimeFormatOptions || state.uiConfig.locale.dateTimeFormatOptions;
              new Intl.DateTimeFormat(localeCode, options);
              return [fetchSiteConfigComplete(config)];
            } catch (e) {
                return [
                  fetchSiteConfigComplete({ ...config, /* use modified fallback values here and remove try from reducer */ }), 
                  showSystemNotification({
                    message: getIntl().formatMessage(configurationMessages.localeError, { message: e.message })
                  })
                ];
              );
            }
          }),
```